packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Permissions/use-scopes-table.ts

```diff
@@ -90,7 +90,12 @@ const useScopesTable = () => {
         })
       );
 
-      return [userScopesGroup, ...resourceScopesGroups, organizationScopesGroup];
+      return [
+        userScopesGroup,
+        ...resourceScopesGroups,
+        // Hide the organization scopes group if there is no organization scopes
+        ...(organizationScopesGroup.data.length > 0 ? [organizationScopesGroup] : []),
```

Comment:
should we use the same logic for resource scopes as well?

Reply:
Based on the API implementation, a resource with empty scopes won't be returned.  An empty resources list response will be filtered.

Reply:
ah, i just read the name wrong